client/notifications/src/panel/suggestions/styles.scss

```diff
@@ -1,14 +1,14 @@
 $base-background-color: $white;
 $list-background: $white;
-$border-color: lighten( $gray, 20% );
+$border-color: var( --color-neutral-100 );
```

Comment:
semi unrelated but these sass variables are a bit scary.. they end up being global once this style loads.